src/libraries/Microsoft.Extensions.Caching.Abstractions/src/Microsoft.Extensions.Caching.Abstractions.ref.cs

```diff
@@ -132,7 +132,7 @@ public partial class PostEvictionCallbackRegistration
         public Microsoft.Extensions.Caching.Memory.PostEvictionDelegate? EvictionCallback { get { throw null; } set { } }
         public object? State { get { throw null; } set { } }
     }
-    public delegate void PostEvictionDelegate(object key, object? value, Microsoft.Extensions.Caching.Memory.EvictionReason reason, object? state);
+    public delegate void PostEvictionDelegate(object key, object value, Microsoft.Extensions.Caching.Memory.EvictionReason reason, object state);
```

Comment:
What about these nullability differences? The reference source marked these parameters as nullable but the source doesn't. @maxkoshevoi @eerhardt